PF/Assignment2/ReverseRightTringleTest.java

```diff
@@ -0,0 +1,52 @@
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReverseRightTringleTest {
```

Comment:
ACTION: follow the comments for the class 'PascalTringleTest' and implement here too.